lnt/tests/test_suite.py

```diff
@@ -581,7 +581,11 @@ def _build(self, path):
                               '-t', target,
                               '-j', str(self._build_threads())] +
                              ([] if self.opts.succinct else ["-v"]) +
-                             ["--"] + shlex.split(self.opts.build_tool_options),
```

Comment:
Can we make --build-tool-options default to "" instead? This way the split should produce an empty list that will concatenate with ["--"] into ["--"].

Reply:
Yes indeed. Done in next version. Thanks for the suggestion